frontend/src/components/SummaryDeclaration.vue

```diff
@@ -357,7 +357,7 @@
                   and conditions. I further confirm that by clicking “I agree” below, I represent and warrant that:
                 </p>
 
-                <ol type="a" style="padding-top: 10px; padding-left: 20px">
+                <ol type="a" style="padding-top: 10px; padding-left: 20px; list-style-position: inside">
```

Comment:
Since we re-use this exact same style in SummaryDec CR - can you make these into a class in common.css so they can be shared across files?